orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -11,310 +11,360 @@ ORT_RUNTIME_CLASS(CheckpointState);  /// Type that holds the training states for
+   *
+   */
   ORT_CLASS_RELEASE(CheckpointState);
 
   /** \brief Export a model that can be used for inferencing.
-  *
-  * If the training session was provided with an eval model, the training session can generate
-  * an inference model if it knows the inference graph outputs. The input inference graph outputs
-  * are used to prune the eval model so that the output model's outputs align with the provided outputs.
-  * The exported model is saved at the path provided and can be used for inferencing with InferenceSession.
-  * Note that the function re-loads the eval model from the path provided to CreateTrainingSession and expects
-  * that this path still be valid.
-  *
-  * \param[in] sess The training session.
-  * \param[in] inference_model_path Path where the inference model should be serialized to.
-  *
-  * \snippet{doc} snippets.dox OrtStatus Return Value
-  *
-  */
+   *
+   * If the training session was provided with an eval model, the training session can generate
+   * an inference model if it knows the inference graph outputs. The input inference graph outputs
+   * are used to prune the eval model so that the output model's outputs align with the provided outputs.
+   * The exported model is saved at the path provided and can be used for inferencing with InferenceSession.
+   * Note that the function re-loads the eval model from the path provided to CreateTrainingSession and expects
+   * that this path still be valid.
+   *
+   * \param[in] sess The training session.
+   * \param[in] inference_model_path Path where the inference model should be serialized to.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
+   *
+   */
   ORT_API2_STATUS(ExportModelForInferencing, _Inout_ OrtTrainingSession* sess,
                   _In_ const ORTCHAR_T* inference_model_path, size_t graph_outputs_len,
                   _In_reads_(graph_outputs_len) const char* const* graph_output_names);
+
+  /** \brief Sets the seed used for random number generation in Onnxruntime.
+   *
+   * Use this to get reproducible results.
+   *
+   * \param[in] seed The seed to be set.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
+   *
+   */
+  ORT_API2_STATUS(SetSeed, _In_ const int64_t seed);
+
+  /** \brief Retrieves the number of user inputs in the training model.
+   *
+   * This function returns the number of inputs of the training model.
+   *
+   * \param[in] sess The training session which has working knowledge of the training model.
+   * \param[out] out Number of user inputs in the training model.
+   *
+   * \snippet{doc} snippets.dox OrtStatus Return Value
+   *
+   */
+  ORT_API2_STATUS(TrainingSessionGetTrainingModelInputCount, _In_ const OrtTrainingSession* sess, _Out_ size_t* out);
+
+  /** \brief Retrieves the number of user inputs in the eval model.
+   *
+   * This function returns the number of inputs of the eval model.
+   *
+   * \param[in] sess The training session which has working knowledge of the eval model.
```

Comment:
Shall we change "The training session which has working knowledge of the eval model." to "The training session which 
 owns the eval model." ?